### DISCUSSION TITLE
Docker shairport-sync configuration file

### DISCUSSION THREAD

**Discussion by simanuel** [mikebrady/shairport-sync#1764] [Q&A] [answered]
Hi, 

I'm trying to use my configuration file in Docker on a raspberry Pi 4 (freshly installed Bookworm), but it's not working. I already tried moving the file into the home folder etc., but it's not working. 

In the readme-file it just says:
> To get access to the full range of configuration options, pass the configuration file to /etc/shairport-sync.conf in the container using the -v option or docker compose.

So I tried this:
`docker run -d --restart unless-stopped --net host --device /dev/snd mikebrady/shairport-sync -v /etc/shairport-sync.conf`

But it's not using the configuration file. 

The answer is probably pretty simple, I'm just doing it wrong - but I have absolutely no idea what's wrong here, so thank you for any help!

**Comment by mikebrady** [comment]
Thanks for the post. The `-v` option mentioned here is a Docker option, not a Shairport Sync option. To be recognized by Docker it should precede the name of the image, so it should go before the `mikebrady/shairport-sync` somewhere. It is used to map a host file into the Docker image — you might have to look up how exactly to use it.

An alternative to using a configuration file might be to pass command line arguments to Shairport Sync, depending on what you want to do.

**Comment by simanuel** [comment, marked as answer]
Thank you very much - as I said, it's probably pretty simple, but I had no clue. It works finally with this command:

`docker run -v /etc/shairport-sync.conf:/etc/shairport-sync.conf -d --restart unless-stopped --net host --device /dev/snd mikebrady/shairport-sync`

Maybe an example like that could be added to the docker readme file to clarify how to use the -v option.
